internal/bminventory/inventory.go

```diff
@@ -102,19 +104,19 @@ func buildHrefURI(base, id string) *strfmt.URI {
 	return strToURI(fmt.Sprintf("%s/%ss/%s", baseHref, base, id))
 }
 
-func (b *bareMetalInventory) monitorImageBuild(ctx context.Context, id string) error {
+func (b *bareMetalInventory) monitorImageBuild(ctx context.Context, jobName string) error {
```

Comment:
it doesn't monitor image build anymore.
```suggestion
func (b *bareMetalInventory) monitorJobProgress(ctx context.Context, jobName string) error {
```